### DISCUSSION TITLE
如何导入祈愿数据

### DISCUSSION THREAD

**Discussion by JadeVane** [PaiGramTeam/PaiGram#207] [Q&A] [answered]
按照提示，进入 https://paimon.moe/wish/import 导入了数据，但是 paimon.moe 并没有提供可用于 paimonbot 导入数据的 url

**Comment by chuangbo** [comment, marked as answer]
按照 paimon.moe 的方法，获取抽卡链接，然后回复给派蒙机器人，并不是导入 paimon.moe
